asteroid/scripts/asteroid_cli.py

```diff
@@ -17,24 +20,19 @@
 ]
 
 
-def upload():
-    """CLI function to upload pretrained models.
-
-    Args:
-        publish_dir (str): Path to the publishing directory.
-            Usually under exp/exp_name/publish_dir
-        uploader (str): Full name of the uploader (Ex: Manuel Pariente)
-        affiliation (str, optional): Affiliation (no accent).
-        git_username (str, optional): GitHub username.
-        token (str): Access token generated to upload depositions.
-        force_publish (bool): Whether to directly publish without
-            asking confirmation before. Defaults to False.
-        use_sandbox (bool): Whether to use Zenodo's sandbox instead of
-            the official Zenodo.
```

Comment:
Removed these docstrings since these aren't actually function params

Reply:
Oh... Didn’t think about that. I will check if there is a better way.

Reply:
<img width="528" alt="Bildschirmfoto 2020-11-19 um 13 10 34" src="https://user-images.githubusercontent.com/175722/99664594-b2d1ee80-2a68-11eb-967f-ad0618f2c893.png">

What do you think about this? It is autogenerated from `--help`.

Or do you want actual Python code docs for people that want to work on the CLI code itself?

Reply:
I think this is reasonable, but it's a bit ugly. 
Anyway, we should probably change the way we make CLI (using `click`, `typer` or `fire` which I discovered only recently), so yes, it does look fine for now.